src/components/utils/RelativeOverlayPortal.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, ReactNode, MutableRefObject, useEffect } from 'react';
+import { useOverlayContainer, useRelativePositioning } from '@equinor/fusion-components';
+import { createPortal } from 'react-dom';
+
+type OverlayProps = {
+    children?: ReactNode;
+    relativeRef: MutableRefObject<HTMLElement | null>;
+    show?: boolean;
+};
+
+const RelativeOverlayPortal: FC<OverlayProps> = ({ children, relativeRef, show }) => {
+    const element = document.createElement('div');
+    const overlayContainer = useOverlayContainer();
+    const rect = useRelativePositioning(relativeRef);
+
+    useEffect(() => {
```

Comment:
Is this neccecary at all? The `element` variable is not used elsewhere in the component. Either remove the `element` and `useEffect` or point the portal to the `element` instead of the `overlayContainer`